### DISCUSSION TITLE
Permission by content Type

### DISCUSSION THREAD

**Discussion by amine2z** [strapi/strapi#5964] [Legacy Topics]
Hello first, thank for all maintainers for this greate job into this. project.
I'm searching in documentation, but dont find, If i have a category content type. theres a way to add a role by this content type. example adding role (find/create...) for only one category.

Thank you for your response in advance

regards

**Comment by derrickmehaffy** [comment]
Hello,

I see you are wanting to ask a question that is not really a bug report or a feature request, questions should be directed to the [discussions](https://github.com/strapi/strapi/discussions)

I have converted this to a discussion for you, we try to keep Github clean and only for bug reports and feature requests.

**Comment by lauriejim** [comment]
Hello!
You will have to follow this Guide.
https://strapi.io/documentation/3.0.0-beta.x/guides/is-owner.html
